app/agreement/create.js

```diff
@@ -6,6 +6,7 @@ const AGREEMENT = {
   agreementNumber: undefined,
   confirmed: false,
   submitted: false,
+  statusId: 1,
```

Comment:
Could we create a data model map for statusIds to eliminate magic numbers?

I have no idea what a statusId or 1 means for e.g.

Reply:
added a map for consistency